testsuite/forge/src/backend/k8s/mod.rs

```diff
@@ -69,19 +78,28 @@ impl Factory for K8sFactory {
     fn versions<'a>(&'a self) -> Box<dyn Iterator<Item = Version> + 'a> {
         Box::new(std::iter::once(Version::new(
             0,
-            "mock-version (k8s does not currently support node versions)".into(),
+            self.image_tag.clone(),
         )))
     }
 
-    fn launch_swarm(&self, _node_num: NonZeroUsize, _version: &Version) -> Result<Box<dyn Swarm>> {
+    fn launch_swarm(&self, node_num: NonZeroUsize, version: &Version) -> Result<Box<dyn Swarm>> {
+        let _ = clean_k8s_cluster(
+            self.helm_repo.clone(),
+            node_num.get(),
+            format!("{}", version),
+            DEFAULT_TESTNET_IMAGE_TAG.to_string(),
```

Comment:
should this be `self.image_tag` instead of using the default testnet tag?

Reply:
I assume testnet tag should be different with node image tag?

Reply:
oh yea sorry I misread this. We should probably use the default testnet tag, you're right.

Reply:
good to have confirming with you which you are expertise in this part